checkov/common/runners/runner_registry.py

```diff
@@ -579,14 +579,15 @@ def print_reports(
 
     def _print_to_console(self, output_formats: dict[str, str], output_format: str, output: str, url: str | None = None) -> None:
         """Prints the output to console, if needed"""
-
         output_dest = output_formats[output_format]
         if output_dest == CONSOLE_OUTPUT:
             del output_formats[output_format]
 
             print(output)
             if url:
+                url = bc_integration.get_sso_prismacloud_url(report_url=url)
```

Comment:
as mentioned before, please invoke your method here https://github.com/bridgecrewio/checkov/blob/2df19723d80024544df918bf85cf73b88e5edec7/checkov/common/bridgecrew/platform_integration.py#L576
the URL comes from this method and is used in 2 areas, so both can benefit from it 🙂

Reply:
@gruebel oh, now I get it. 👍